src/components/atoms/UserStatusDropdown/UserStatusDropdown.tsx

```diff
@@ -1,39 +1,47 @@
-import React, { useMemo } from "react";
+import React, { useMemo, useEffect } from "react";
 import { Dropdown, DropdownButton } from "react-bootstrap";
 
-import { USER_STATUSES } from "settings";
+import { UserStatus } from "types/User";
 
 import { useProfileStatus } from "hooks/useProfileStatus";
 
-import { UserStatus } from "types/User";
-
 import "./UserStatusDropdown.scss";
 
 export interface UserStatusDropdownProps {
   userStatuses?: UserStatus[];
 }
 
 export const UserStatusDropdown: React.FC<UserStatusDropdownProps> = ({
-  userStatuses,
+  userStatuses = [],
 }) => {
   const { status, changeUserStatus } = useProfileStatus();
 
-  const allUserStatuses = useMemo(
-    () => (userStatuses ? [...USER_STATUSES, ...userStatuses] : USER_STATUSES),
-    [userStatuses]
-  );
+  // This will check if the user status from the database exists in the venue user statuses and if it doesn't, it will fallback to the first one from the list.
+  useEffect(() => {
+    const hasUserStatuses = !!userStatuses.length;
```

Comment:
I believe we can move this check to `src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx`
and render `UserStatusDropdown` only when the statuses are enabled and the length of the statuses array > 0.

Reply:
I'll check and apply it if this is true :)

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1772/commits/b2fbb77490f27b007d651eb4d75d2e62d3fb15e6